interpreter/core/computer/ai/ai.py

```diff
@@ -106,9 +106,10 @@ def query(self, text, query, custom_reduce_query=None):
         # Retrieve OpenAI API key
         openai_api_key = os.getenv("OPENAI_API_KEY")
         if openai_api_key is None:
-            openai_api_key = getpass.getpass(
-                prompt="Please enter your OpenAI API key: "
+            openai_api_key = prompt(
+                message="Please enter your OpenAI API key: ", is_password=True
```

Comment:
Does it need the `message` key?
 https://python-prompt-toolkit.readthedocs.io/en/master/pages/asking_for_input.html#password-input

Reply:
¨no